api/pkg/app/gorm_logger.go

```diff
@@ -69,7 +69,7 @@ func (w *prodWriter) Printf(format string, data ...interface{}) {
 				log = log.With("unknown", d)
 			}
 		default:
-			log = log.With("unknown", d)
+			log = log.With("unknown", dt)
```

Comment:
Should not this be `d` ?

Reply:
Then it would give a compile error of `dt` not used

Reply:
i guess it should be `d` ..`dt` would be the type which is not required in the log

Reply:
Would it be fine if we log as `log = log.With(dt, d)`

Reply:
Since it is just a type

Reply:
You can use

```
switch d.(type) 
```

Then you don't require `_ = dt`

Reply:
This would give `golangci-lint` error